src/families/polkadot/js-signOperation.ts

```diff
@@ -52,6 +53,10 @@ const getExtra = (type: string, account: Account, transaction: Transaction) => {
       }
     : {};
 
+  if (transaction.mode == "send" && transaction.useAllAmount) {
+    extra.palletMethod = MODE_TO_PALLET_METHOD["sendMax"];
+  }
+
```

Comment:
Code readability : can you please rework the `const extra = ...` definition above to merge it with this?
You can rely on if/else as you did here, I think it's way more readable than the multiple ternary operators above.